cmd/forwarder/root.go

```diff
@@ -19,13 +20,16 @@ import (
 const envPrefix = "FORWARDER"
 
 func rootCommand() *cobra.Command {
+	var configFile string
```

Comment:
Let's remove this variable, since it is not used anywhere else.

Reply:
Sure.

Reply:
bind.ConfigFile(cmd.PersistentFlags(), new(string))